src/platform/packages/private/kbn-scout-reporting/src/reporting/report/events/persistence/index.ts

```diff
@@ -108,30 +108,37 @@ export class ScoutReportDataStream {
     await this.es.index({ index: SCOUT_TEST_EVENTS_DATA_STREAM_NAME, document: event });
   }
 
-  async addEventsFromFile(eventLogPath: string) {
-    // Make the given event log path absolute
-    eventLogPath = path.resolve(eventLogPath);
-
-    const events = async function* () {
-      const lineReader = readline.createInterface({
-        input: fs.createReadStream(eventLogPath),
-        crlfDelay: Infinity,
-      });
-
-      for await (const line of lineReader) {
-        yield line;
+  async addEventsFromFile(...eventLogPaths: string[]) {
+    // Normalize the given event log paths to absolute paths
+    const paths: string[] = eventLogPaths.map((p) => path.resolve(p));
+
+    const events = async function* (): AsyncGenerator<string> {
+      for (const filePath of paths) {
+        const lineReader = readline.createInterface({
+          input: fs.createReadStream(filePath),
+          crlfDelay: Infinity,
+        });
+
+        for await (const line of lineReader) {
+          yield line;
+        }
       }
     };
 
     this.log.info(
-      `Uploading events from file ${eventLogPath} to data stream '${SCOUT_TEST_EVENTS_DATA_STREAM_NAME}'`
+      [
+        `Uploading events from ${paths.length} file(s) to data stream '${SCOUT_TEST_EVENTS_DATA_STREAM_NAME}':`,
+        ...paths.map((filePath) => `- ${filePath}`),
+      ].join('\n')
     );
 
     const stats = await this.es.helpers.bulk({
       datasource: events(),
       onDocument: () => {
         return { create: { _index: SCOUT_TEST_EVENTS_DATA_STREAM_NAME } };
       },
+      refresh: false,
+      refreshOnCompletion: true,
```

Comment:
@dgieselaar I think you're right and it's not necessary to trigger an index refresh after we upload the events. It adds unnecessary load on the target ES cluster, so we should disable it.

```suggestion
      refreshOnCompletion: false,
```

Alternatively, you can surface this as a flag to the Scout CLI and default to "no refresh". Either way is fine by me. 👍🏽